content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Submitting blog posts and case studies
+linktitle: Blogs and case studies
+slug: blogs-case-studies
+content_template: templates/concept
+weight: 30
+---
+
+
+{{% capture overview %}}
+
+Anyone can write a blog post and submit it for review. You must submit case studies
+to a committee for approval.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Write a blog post
+
+Blog posts should not be
```

Comment:
Is there separate documentation for the Blog sub-project (there is probably a readme or slack channel, etc)? If so, link to it here or create another section from the base contributing section.